docs/guide/acquiring_data/helioviewer.rst

```diff
@@ -18,64 +18,104 @@ install Glymur on your system.
+    ...     print(sorted_ids)  # doctest: +REMOTE_DATA
+    (('SOHO', 'EIT', None, '171'), 0)
+    (('SOHO', 'EIT', None, '195'), 1)
+    (('SOHO', 'EIT', None, '284'), 2)
+    (('SOHO', 'EIT', None, '304'), 3)
+    (('SOHO', 'LASCO', 'C2', 'white-light'), 4)
+    (('SOHO', 'LASCO', 'C3', 'white-light'), 5)
+    (('SOHO', 'MDI', None, 'magnetogram'), 6)
+    (('SOHO', 'MDI', None, 'continuum'), 7)
+    (('SDO', 'AIA', None, '94'), 8)
+    (('SDO', 'AIA', None, '131'), 9)
+    (('SDO', 'AIA', None, '171'), 10)
+    (('SDO', 'AIA', None, '193'), 11)
+    (('SDO', 'AIA', None, '211'), 12)
+    (('SDO', 'AIA', None, '304'), 13)
+    (('SDO', 'AIA', None, '335'), 14)
+    (('SDO', 'AIA', None, '1600'), 15)
+    (('SDO', 'AIA', None, '1700'), 16)
+    (('SDO', 'AIA', None, '4500'), 17)
+    (('SDO', 'HMI', None, 'continuum'), 18)
+    (('SDO', 'HMI', None, 'magnetogram'), 19)
+    (('STEREO_A', 'SECCHI', 'EUVI', '171'), 20)
+    (('STEREO_A', 'SECCHI', 'EUVI', '195'), 21)
+    (('STEREO_A', 'SECCHI', 'EUVI', '284'), 22)
+    (('STEREO_A', 'SECCHI', 'EUVI', '304'), 23)
+    (('STEREO_B', 'SECCHI', 'EUVI', '171'), 24)
+    (('STEREO_B', 'SECCHI', 'EUVI', '195'), 25)
+    (('STEREO_B', 'SECCHI', 'EUVI', '284'), 26)
+    (('STEREO_B', 'SECCHI', 'EUVI', '304'), 27)
+    (('STEREO_A', 'SECCHI', 'COR1', 'white-light'), 28)
+    (('STEREO_A', 'SECCHI', 'COR2', 'white-light'), 29)
+    (('STEREO_B', 'SECCHI', 'COR1', 'white-light'), 30)
+    (('STEREO_B', 'SECCHI', 'COR2', 'white-light'), 31)
+    (('PROBA2', 'SWAP', None, '174'), 32)
+    (('Yohkoh', 'SXT', None, 'AlMgMn'), 33)
+    (('Yohkoh', 'SXT', None, 'thin-Al'), 34)
+    (('Yohkoh', 'SXT', None, 'white-light'), 35)
+    (('Hinode', 'XRT', 'Al_med', 'Al_thick'), 39)
+    (('Hinode', 'XRT', 'Al_med', 'Be_thick'), 40)
+    (('Hinode', 'XRT', 'Al_med', 'Open'), 42)
+    (('Hinode', 'XRT', 'Al_med', 'Ti_poly'), 43)
+    (('Hinode', 'XRT', 'Al_poly', 'Al_mesh'), 44)
+    (('Hinode', 'XRT', 'Al_poly', 'Al_thick'), 45)
+    (('Hinode', 'XRT', 'Al_poly', 'Be_thick'), 46)
+    (('Hinode', 'XRT', 'Al_poly', 'Open'), 48)
+    (('Hinode', 'XRT', 'Al_poly', 'Ti_poly'), 49)
+    (('Hinode', 'XRT', 'Be_med', 'Open'), 54)
+    (('Hinode', 'XRT', 'Be_thin', 'Open'), 60)
+    (('Hinode', 'XRT', 'C_poly', 'Al_mesh'), 62)
+    (('Hinode', 'XRT', 'C_poly', 'Al_thick'), 63)
+    (('Hinode', 'XRT', 'C_poly', 'Open'), 66)
+    (('Hinode', 'XRT', 'C_poly', 'Ti_poly'), 67)
+    (('Hinode', 'XRT', 'Open', 'Al_mesh'), 69)
+    (('Hinode', 'XRT', 'Open', 'Al_thick'), 70)
+    (('Hinode', 'XRT', 'Open', 'Be_thick'), 71)
+    (('Hinode', 'XRT', 'Open', 'Ti_poly'), 74)
+    (('TRACE', '171', None, None), 75)
+    (('TRACE', '195', None, None), 76)
+    (('TRACE', '284', None, None), 77)
+    (('TRACE', '1216', None, None), 78)
+    (('TRACE', '1550', None, None), 79)
+    (('TRACE', '1600', None, None), 80)
+    (('TRACE', '1700', None, None), 81)
```

Comment:
I think the way the dictionary is created needs to be changed for Trace. 
`('TRACE', '171', None, None)` should be `('TRACE', None, None, '171')`